src/common/Coverage.ts

```diff
@@ -21,12 +21,67 @@ import { CDPSession } from './Connection';
 
 import { EVALUATION_SCRIPT_URL } from './ExecutionContext';
 
+/**
+ * The CoverageEntry class represents one entry of the coverage report.
+- `url` <[string]> StyleSheet URL
+- `text` <[string]> StyleSheet content
+- `ranges` <[Array]<[Object]>> StyleSheet ranges that were used. Ranges are sorted and non-overlapping.
+  - `start` <[number]> A start offset in text, inclusive
+  - `end` <[number]> An end offset in text, exclusive
+*/
+
 interface CoverageEntry {
+  /**
+   * The URL of the style sheet or script.
+   */
   url: string;
+  /**
+   * The content of the style sheet or script.
+   */
   text: string;
+  /**
+   * The covered range as start and end positions.
+   */
   ranges: Array<{ start: number; end: number }>;
 }
 
+/**
+ * The Coverage class provides methods to gathers information about parts of
+ * JavaScript and CSS that were used by the page.
+ * 
+ * @example
```

Comment:
I think the `@remark` should go before the example so the summary (the stuff pre-remark) is nice and concise.

Reply:
Done.